game/templates/game/level_moderation.html

```diff
@@ -110,12 +110,13 @@
 
 {% if levelData %}
   <div class="tableWrapper">
-    {% if thead %}
     <table id="" class="CSSTable wide alternate smaller">
       <tr>
-      {% for header in thead %}
-        <th> {{header}} </th>
-      {% endfor %}
+        <th>{% trans "Student" %}</th>
+        <th>{% trans "Level name" %}</th>
+        <th>{% trans "Shared with" %}</th>
+        <th>{% trans "Play" %}</th>
+        <th>{% trans "Delete" %}</th>
```

Comment:
Yes, I was thinking about that, but I in my opinion headers fit more with the view, at least in this case.